### DISCUSSION TITLE
https://github.com/ only in DarkMode<title>

### DISCUSSION THREAD

**Discussion by JohannesPaulHoffmann** [community/community#81023] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi,

when I visit the main site: https://github.com/ it's only in DarkMode, hard to see, can there be a button?

The rest is then in a 'BrightMode'.

Regards,
Johannes

**Comment by leeringrose** [comment]
I am sure you should be able to configure your appearance on your profile setting.

**Comment by JohannesPaulHoffmann** [comment]
@leeringrose no, it's in the System/Webbrowser, but github.com's main page doesn't change at all, but the Login does.

:-Þ.

**Comment by SIGMazer** [comment]
@JohannesPaulHoffmann  you can change it from here `https://github.com/settings/appearance` and make sure that you don't have any browser extension the effect the appearance

**Comment by JohannesPaulHoffmann** [comment]
It is already "Light default", what I mean is the github.com side, espacially when I Sign Out, but I can still browse GitHub in a "Light Mode", as My Firefox Browser is set to that, but this one side (probably Sign Out for that): github.com is "Dark Mode", which can't be changed.

**Comment by timred01** [comment]
Download & install extension dark reader, hoho

**Comment by ashkan0201** [comment]
As an AI language model, I don't have direct control over website functionalities or appearance. However, I can suggest a general solution for switching between Dark Mode and Bright Mode on GitHub.

GitHub does not have a built-in toggle button to switch between Dark Mode and Bright Mode on its main site. However, you can use browser extensions or settings to enable Dark Mode for GitHub or customize the appearance.

Here's how you can achieve this using some popular web browsers:

**Google Chrome:**
1. Install a Dark Mode extension from the Chrome Web Store, such as "Dark Reader" or "Night Eye".
2. Once the extension is installed, it should automatically enable Dark Mode for GitHub and other websites. You can customize the extension's settings based on your preferences.

**Mozilla Firefox:**
1. Install a Dark Mode extension from the Firefox Add-ons store, such as "Dark Reader" or "Dark Mode".
2. After installing the extension, it should apply Dark Mode to GitHub and other websites. You may also have customization options available.

Please note that the specific extensions and their functionalities may vary, so you can explore different options or search for other extensions that suit your needs. Additionally, browser updates or changes in GitHub's website design could affect the functionality of these extensions.

If you prefer to have a built-in toggle button on the GitHub website itself, you can provide feedback to GitHub by following these steps:

1. Scroll to the bottom of the GitHub website.
2. Click on "Provide feedback" or "Contact GitHub" (the exact wording may differ).
3. Fill out the feedback form and express your request for a Dark Mode toggle button on the main site.

By providing feedback directly to GitHub, you can help them understand user preferences and potentially influence future updates to their website.

Remember that these suggestions are based on the current state of GitHub as of my knowledge cutoff in September 2021. It's possible that GitHub may have introduced Dark Mode or made changes to their site since then.

**Comment by JohannesPaulHoffmann** [comment]
Guys, the main page is in dark mode (see the 
![Screenshot_20240119-214947](https://github.com/community/community/assets/110303329/7589ef58-b89d-4b67-8f9a-d3e153e9acee)
picture attached), while (basically) 'everythin' else', like when I click to Login, is in white mode.

P. S.: Didn't know it would 'attach' the picture like that, well stays now like that (as I write from mobile).

**Reply by JohannesPaulHoffmann** [reply]
Never mind My "'P. S.: […]", as I didn't know that all this inserted text, 'attachment', does display as a picture (when I sent it).
